modules/ROOT/pages/create-community.adoc

```diff
@@ -244,6 +249,11 @@ These permissions reflect the current API Community Manager package version. If
 
 == Configure Sharing Settings
 
+Configure the settings with one of these options:
+
+. Configure sharing settings for members and guests automatically with default settings with the *Fix* buttons in the xref:diagnostics.adoc#user-access-tab[API Community Manager diagnostics user access tab].
+. Configure sharing settings for members and guests manually with custom settings with these steps.
```

Comment:
ditto previous: not numbered, clarify "these" steps.

Reply:
Fixed.